### ISSUE TITLE
Use current desktop wallpaper in Desktop View?

### ISSUE THREAD

**Issue by samwzlim** [mzomparelli/zVirtualDesktop#273]
Is it possible to show the current desktop wallpaper in desktop view? I have no intentions of using separate backgrounds for different virtual desktops, so I want a translucent background (sort of how native task view looks like)

**Comment by mzomparelli** [comment]
So you want your wallpaper to be the full background for desktopview or do you want desktopview to have transparency? With transparency it might not be your desktop that is behind desktopview.

**Comment by samwzlim** [comment]
> So you want your wallpaper to be the full background for desktopview or do you want desktopview to have transparency? With transparency it might not be your desktop that is behind desktopview.

**Comment by samwzlim** [comment]
@mzomparelli Yeah, the first option is what I'm looking for, maybe Desktop View can take the current desktop wallpaper and set is as the background. Maybe add an option for opacity as well? Task View in Windows 10 shows the desktop wallpaper, but it's got a lower opacity for better visibility of the windows currently on that desktop

**Comment by mzomparelli** [comment]
Thanks for the feedback. This seems like an easy enhancement. I'll take a look at this next time I'm in the code.
